config/all-in-one/templates/cp4waiops-custom.yaml

```diff
@@ -25,8 +25,8 @@ spec:
       project: default
       source:
         path: config/cp4waiops-custom
-        repoURL: https://github.com/IBM/cp4waiops-gitops
-        targetRevision: HEAD
+        repoURL: https://github.com/morningspace/cp4waiops-gitops
```

Comment:
why using morningspace?

Reply:
It's just because I'm using it to do the local verification. Once it passes review, I will change it back to use IBM org. Ultimately, I think we may need this to be parameterized, so that user can overwrite w/o hard coding it in YAML.